che-theia-plugins.yaml

```diff
@@ -130,6 +162,10 @@ plugins:
       revision: v1.5.0
     sidecar:
       directory: java8
+      memoryLimit: "1500Mi"
+      volumeMounts:
+      - name: m2
+        mount: "/home/theia/.m2"
```

Comment:
I think it should be `path`

Reply:
Yes I messed up my find/replace :smiley: